docs/standard-library.rst

```diff
@@ -22,34 +22,63 @@ It behaves exactly like the generic :class:`structlog.BoundLogger` except:
 Processors
 ----------
 
-structlog comes with one standard library-specific processor:
+structlog comes with a few standard library-specific processors:
 
 :func:`~structlog.stdlib.filter_by_level`:
-   Checks the log entries's log level against the configuration of standard library's logging.
+   Checks the log entry's log level against the configuration of standard library's logging.
    Log entries below the threshold get silently dropped.
    Put it at the beginning of your processing chain to avoid expensive operations happen in the first place.
 
+:func:`~structlog.stdlib.add_logger_name`:
+   Adds the name of the logger to the output under the key ``logger``.
```

Comment:
I guess I’m being pedantic here, but I think it should be “to the event dictionary” to be clear.